app/js/components/demographics.jsx

```diff
@@ -17,249 +18,361 @@ export default class Demographics extends React.Component {
       uuid: null,
       gender: '',
       birthdate: '',
-      birthDateEstimated: false,
+      birthDateTime: '',
+      birthdateEstimated: false,
       deceased: false,
       deathDate: '',
+      finalDeathdate: '',
       deathDateEstimated: false,
       causeOfDeath: [],
       nameOfCauseOfDeath: '',
       createdBy: '',
       dateCreated: '',
-      deleted: false
+      deleted: false,
+      birthdateError: '',
+      deathDateError: '',
+      causeOfDeathError: '',
+      format: 'YYYY-MM-DD HH:mm:ss',
+      inputFormat: 'YYYY-MM-DD',
+      mode: 'date',
+      deathDateTime: ''
     };
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSave = this.handleSave.bind(this);
     this.onDeceased = this.onDeceased.bind(this);
     this.setGender = this.setGender.bind(this);
-    this.setBirthDateEstimate = this.setBirthDateEstimate.bind(this);
+    this.setbirthdateEstimate = this.setbirthdateEstimate.bind(this);
     this.setDeathDateEstimate = this.setDeathDateEstimate.bind(this);
     this.handleTimeChange = this.handleTimeChange.bind(this);
   }
 
-  /**
-   *
-   * set to state patient attributes
-   *
-   * @memberOf Demographics
-   */
-  componentWillMount() {
+
+  componentDidMount() {
     this.state.uuid = this.props.uuid;
-    apiCall(null,'get',`/patient/${this.state.uuid}?v=full`)
-      .then((res) => {
-        const { auditInfo, gender, birthdate, birthDateEstimated, dead: deceased, causeOfDeath, deathDate, deathDateEstimated, voided: deleted } = res.person
+    apiCall(null, 'get', `/patient/${this.state.uuid}?v=full`).
+      then((response) => {
+        const { auditInfo, gender, birthdate: birthDateTime, birthdateEstimated,
+                dead: deceased,
+                causeOfDeath,
+                deathDate: deathDateTime,
+                deathDateEstimated,
+                voided: deleted } = response.person
         const { display: createdBy } = auditInfo.creator
         const dateCreated = auditInfo.dateCreated
+        const birthdate = moment(birthDateTime).format('YYYY-MM-DD HH:mm:ss')
+        const finalBirthdate = moment(birthdate).
+                               format('YYYY-MM-DDTHH:mm:ss.000Z')
```

Comment:
Any reason why you brought this to a new line?

Reply:
@kingisaac95  it would go beyond the 80 chr line length if put in the first line